### PR TITLE
Initial version of the graph concept docs

### PR DESCRIPTION
**Pull request by ikas**
An initial version of the graph concept docs. Might require some tweaking after the graph endpoint docs.

Related to [GTC-729](https://gfw.atlassian.net/browse/GTC-729).

### CODE REVIEW

File: source/includes/concept/_graph.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## Graph
+
+As you have read in the [Dataset concept section](/index-rw.html#dataset), one of the main goals of the RW API is to provide a common interface for interacting with data provided by a variety of services. However, given the number of datasets currently hosted in the RW API (2506 at the time of writing), it can be hard to find exactly what dataset you are looking for. It can even be overwhelming to navigate through the datasets list, due to the number of datasets available and the wide range of topics each dataset relates to.
+
+The RW API provides you different ways to search for the information you are looking for: from [searching datasets](/index-rw.html#search) by keyword and exploring [dataset metadata](/index-rw.html#metadata12), to categorization through the use of [vocabulary and tags](/index-rw.html#vocabulary-and-tags). However, none of these options is optimal when it comes to finding similar datasets, related to the ones you find relevant. This is where the Graph service comes to the rescue.
```
Comment (tiagojsag) [inline source/includes/concept/_graph.md]: remove the /index-rw.html from the links please

**Review by tiagojsag — CHANGES_REQUESTED** [review]

**Review by ethanroday — CHANGES_REQUESTED** [review]
All the graph stuff happens automatically under the hood, right? I would add some language that basically says, "this is stuff you get for free when you upload/tag a dataset and is mostly read-only". Otherwise looks good!

**Comment by ikas** [comment]
Hey Ethan - users have to explicitly define the graph relationships for their datasets. Graph endpoint docs were severely incomplete and most creation/edition/deletion endpoints were not documented. So, the answer to your question is no, this is not at all automatic - we'll explain how users can do it in the endpoint docs. 

Do you think, it's worth adding a note here highlighting that?

**Comment by ethanroday** [comment]
Ah, got it. In that case, LGTM! 🚀

**Review by ethanroday — APPROVED** [review]

**Pull request merged**